src/generic/ResidueField.jl

```diff
@@ -668,3 +668,14 @@ end
 function  gen(R::AbstractAlgebra.Generic.ResField{AbstractAlgebra.Generic.Poly{Rational{BigInt}}})
    return R(gen(base_ring(R)))
 end
+
+function RandomExtensions.make(S::AbstractAlgebra.ResField{AbstractAlgebra.Generic.Poly{Rational{BigInt}}}, vs...)
+   R = base_ring(S)
+   if length(vs) == 1 && elem_type(R) == Random.gentype(vs[1])
+      Make(S, vs[1])
+   else
+      n = degree(S.modulus)
+      make(S, make(base_ring(S), 0:n - 1, vs...))
```

Comment:
So in Nemo, this uses `n-1:n-1`, I guess this should also be changed to `0:n-1` there ?

Reply:
Oh, does it? I don't understand the second flag then:
```
julia> f = x^2 + 5x - 8
x^2 + 5*x - 8

julia> K, a = NumberField(f, "a", cached = false)
(Number field over Rational Field with defining polynomial x^2 + 5*x - 8, a)

julia> rand(K, -1:1)
-a + 1

julia> rand(K, -1:1)
1
```

Reply:
I thought that the second argument for the polynomial ring is the target degree?

Reply:
I will change it to be in line with Nemo, but the `rand` behavior is a bit confusing for polynomial rings.

Reply:
> I thought that the second argument for the polynomial ring is the target degree?

The first argument is the target degree, the remaining ones for randomizing coefficients. So in this particular case, the goal was to set the degree range automatically. But I agree that it can be slightly confusing: the target degree is chosen randomly within the target range that is passed. So if you pass `n-1:n-1`, you will get a degree of `n-1` (unless of course when the leading coeff happens to be null), i.e. no randomness is involved for choosing the degree.